lib/tpm2_policy.c

```diff
@@ -157,17 +158,18 @@ static bool tpm2_policy_pcr_build(TSS2_SYS_CONTEXT *sapi_context,
     TPM2B_DIGEST pcr_digest = TPM2B_TYPE_INIT(TPM2B_DIGEST, buffer);
     TPMI_ALG_HASH auth_hash = tpm2_session_get_authhash(policy_session);
 
-    TSS2_RC rval = tpm_hash_sequence(sapi_context, auth_hash, TPM2_RH_NULL,
```

Comment:
Couldn't this function be removed now from lib/tpm_hash.c? There's no other user, or do you prefer to keep it around in case is used in the future?